Signal/translations/de.lproj/Localizable.strings

```diff
@@ -230,10 +230,10 @@
 "ATTACHMENT_ERROR_INVALID_FILE_FORMAT" = "Anhang besitzt ein ungültiges Dateiformat.";
 
 /* Attachment error message for attachments without any data */
-"ATTACHMENT_ERROR_MISSING_DATA" = "Anhang ist leer.";
+"ATTACHMENT_ERROR_MISSING_DATA" = "Der Anhang ist leer.";
 
 /* Accessibility hint describing what you can do with the attachment button */
-"ATTACHMENT_HINT" = "Zu sendende Medieninhalte auswählen";
+"ATTACHMENT_HINT" = "Medieninhalte zum versenden auswählen";
```

Comment:
"Medieninhalte zum Versenden auswählen" would be correct. Words after 'zum' are starting almost always with a uppercase letter!